appinventor/lib/blockly/src/core/xml.js

```diff
@@ -347,27 +350,6 @@ Blockly.Xml.domToBlockInner = function(workspace, xmlBlock, opt_reuseBlock) {
     block.initSvg();
   }
 
-  var inline = xmlBlock.getAttribute('inline');
```

Comment:
@halatmit This code was in the original blockly (1752) that we are using (it may have since changed, but I haven't checked). Lyn moved it further down (see his comment on/near line 459). However he didn't delete it from its original position, which is an error.